src/cloud_provider/service.rs

```diff
@@ -365,30 +368,11 @@ pub fn deploy_user_stateless_service<T>(target: &DeploymentTarget, service: &T)
 where
     T: Service + Helm,
 {
-    deploy_stateless_service(
-        target,
-        service,
-        service.engine_error(
-            EngineErrorCause::User(
-                "Your application has failed to start. \
-                Ensure you can run it without issues with `qovery run` and check its logs from the web interface or the CLI with `qovery log`. \
-                This issue often occurs due to ports misconfiguration. Make sure you exposed the correct port (using EXPOSE statement in Dockerfile or via Qovery configuration).",
-            ),
-            format!(
-                "{} {} has failed to start ⤬",
-                service.service_type().name(),
-                service.name_with_id()
-            ),
-        ),
-    )
+    deploy_stateless_service(target, service)
```

Comment:
Did you forget to handle error?

Reply:
it is already the case, there is no ; so it return the result of deploy_stateless_service, which is the same type that the function retrun;
So it just propagate the error here